actions/get_dependencies/get_dependencies.sh

```diff
@@ -156,6 +156,26 @@ get_lapacke() {
   ${APT_COMMAND} update
   ${APT_GET_COMMAND} install liblapacke liblapacke-dev
 }
+# Wraps installing libfort
+#
+# Usage:
+#   get_libfort
+get_libfort() {
+  if [ -z "${CACHE_LIBFORT}" ]; then
+    export INSTALL_PATH=`pwd`/install
+    wget --no-check-certificate --content-disposition https://codeload.github.com/seleznevae/libfort/tar.gz/refs/tags/v0.4.2
+    tar -zxf libfort-0.4.2.tar.gz
+    cd libfort-0.4.2
+    export CXX=`which g++`
+    export CC=`which gcc`
+    ../cmake-3.16.3-Linux-x86_64/bin/cmake -H. -Bbuild -DCMAKE_INSTALL_PREFIX=${INSTALL_PATH} -DCMAKE_CXX_COMPILER=${CXX} -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_FLAGS="-std=c++17" -DBUILD_SHARED_LIBS=ON -DCMAIZE_GITHUB_TOKEN=$CMAIZE_GITHUB_TOKEN
```

Comment:
You shouldn't need all these flags to install libfort.

Reply:
I just followed libint's build. Let me try to build without these flags.